README.md

```diff
@@ -41,6 +57,27 @@ If you want to find a record using it's slug, a custom finder is provided by the
 $example = $this->Examples->find('slugged', ['slug' => $slug]);
 ```
 
+## Configuration
+
+Slug comes with the following configuration options:
+
+- **field:** name of the field (column) to hold the slug. Defaults to `slug`.
+- **displayField:** name of the field(s) to build the slug from. Defaults to
+     the `\Cake\ORM\Table::displayField()`.
+- **separator:** defaults to `-`.
+- **replacements:** hash of characters (or strings) to custom replace before
+ generating the slug.
+- **maxLength:** maximum length of a slug. Defaults to the field's limit as
+ defined in the schema (when possible). Otherwise, no limit.
+- **slugger:** class that implements the `Muffin\Slug\SlugInterface`. Defaults
+ to `Muffin\Slug\Slugger\CakeSlugger`.
+- **unique:** tells if slugs should be unique. Set this to a callable if you
+ want to customize how unique slugs are generated. Defaults to `true`.
+- **scope:** extra conditions used when checking a slug for uniqueness.
+- **implementedEvents:** events this behavior listens to.
+- **implementedFinders:** custom finders implemented by this behavior.
+- **implementedMethods:** mixin methods directly accessible from the table.
+
```

Comment:
Love that! Should it be commented out as code so it could be re-used in the docblock as is (ala cakephp docblocks description of options). What do you think?